### PR TITLE
[NET_HANDLER] Removed protobuf_tests; updated README and Makefile

### LINKED ISSUE #132: [MISC] Remove protobuf_tests directory and add to Protobuf wiki
The files in the `protobuf_tests` directory contain code to demonstrate that protobufs can be used in net handler. (They don't add functionality to Runtime.) We can add some snippets of the code to the Protobuf wiki to explain how we use protobufs in Runtime.

### PR DESCRIPTION
**Pull request by benliao1**
This is a small PR that completes a small code cleanup ticket:

- Removed `protobuf_tests` folder in `net_handler` and moved to protobuf Wiki page as a tutorial on how to use protobufs
- Updated `net_handler` `README ` and `net_handler` `Makefile` to reflect changes
- Fixed some incorrect and outdated information in the top-level `README` regarding `net_handler`, the addition of a test framework, and the addition of `ncurses` as a library (awaiting @levincent06 's PR)

Closes #132 .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +79,12 @@ Runtime has the following third-party library dependencies:
 	1. Install `doxygen`: `sudo apt-get -y install doxygen`
 	2. Repeat steps 5.i and 5.ii from above in the `protobuf-c` directory, then do `make html`
 	3. Then navigate to that directory in Explorer/Finder and open the `html/index.html` file by double clicking on it
+	
+#### Installing `ncurses`
+
+The command below should install on Raspbian / Debian machines:
+
+1. `sudo apt-get install libncurses5-dev libncursesw5-dev`
```
Comment (levincent06) [inline README.md:88]: I also added a similar section on my shm UI branch! Not sure what steps to take to resolve the potential conflict

**Review by levincent06 — APPROVED** [review]
Just a small comment about adding ncurses to the root README.

File: net_handler/README.md [hunk 2]
```diff
@@ -1,15 +1,14 @@
 # Network Handler
 
-The network handler's job is to communicate with Dawn and Shepherd through sockets and turn the network messages into shared memory updates. Learn more about how the network handler works in our [wiki](https://github.com/pioneers/c-runtime/wiki/Network-Handler).
+The network handler's job is to communicate with Dawn and Shepherd through sockets and turn the network messages into shared memory updates. Learn more about how the network handler works in our [wiki](https://github.com/pioneers/c-runtime/wiki/Network-Handler). For a definition of, resources for, and tutorial on Google protobufs and how they are used in `net_handler`, check out the [Protobuf wiki page](https://github.com/pioneers/runtime/wiki/Protobufs).
```
Comment (AVSurfer123) [inline net_handler/README.md:3]: the added sentence sounds weird to me. I think you can just simplify it to `For tutorials and resources on Google protobufs`

**Review by AVSurfer123 — APPROVED** [review]

**Comment by AVSurfer123** [comment]
The wiki page should give the command for creating the protobuf definitions as well. or at least says to do `make` in the `net_handler` directory

File: README.md (hunk 1 above)
Comment (benliao1) [reply README.md:88]: Ah ok if that's the case I'll revert these changes and the changes from your branch should be added--makes more sense~~
I thought that yours would get merged in soon and I didn't remember seeing it here, which is why I added this, my bad!

**Comment by benliao1** [comment]
(The above force pushes were rebasing onto the `master` from previous PR merge as well as fixing addressing the comments)

**Pull request merged**